docs/2.7.0/docs/canton/usermanual/monitoring.rst

```diff
@@ -326,35 +353,29 @@ Health Checks
+Each Canton node can optionally be configured to start a gRPC server exposing the `gRPC Health Service <https://github.com/grpc/grpc/blob/master/doc/health-checking.md#service-definition>`__. Passive nodes (see :ref:`High Availability <ha_user_manual>` for more information on active/passive states) return ``NOT_SERVING``. Consider this when configuring `liveness and readiness probes in a Kubernetes environment <https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/>`__.
 
-The precise way the state is computed is subject to change and will likely evolve as we refine canton self monitoring capabilities.
+The precise way the state is computed is subject to change.
 
-Example monitoring configuration to be placed inside a node configuration object:
+Here is an example monitoring configuration to place inside a node configuration object:
 
 .. literalinclude:: /canton/includes/mirrored/enterprise/integration-testing/src/main/resources/include/health-monitoring.conf
 
 .. note::
 
-    The gRPC health server is configured per canton node, not per process, as is the case for the HTTP health check server (see below).
-    This means the configuration must be inserted within a node's configuration object.
+    The gRPC health server is configured per Canton node, not per process, as is the case for the HTTP health check server (see below). This means that the configuration must be inserted within a node's configuration object.
 
 .. note::
 
     To support usage as a Kubernetes liveness probe, the health server exposes a service named ``liveness`` that should be targeted when `configuring a gRPC probe <https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-grpc-liveness-probe>`__.
-    The latter service will always return ``SERVING``.
+    The latter service always returns ``SERVING``.
 
 HTTP Health Check
 ~~~~~~~~~~~~~~~~~
 
-The ``canton`` process can optionally expose an HTTP endpoint indicating if the process believes it is healthy.
-This may be used as an uptime check or as a `Kubernetes liveness probe <https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/>`__.
-If enabled, the ``/health`` endpoint will respond to a ``GET`` HTTP request with a 200 HTTP status code if healthy or 500 if unhealthy (with a plain text description of why it is unhealthy).
+The ``canton`` process can optionally expose an HTTP endpoint indicating whether the process believes it is healthy. This may be used as an uptime check or as a `Kubernetes liveness probe <https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/>`__. If enabled, the ``/health`` endpoint will respond to a ``GET`` HTTP request with a 200 HTTP status code (if healthy) or 500 (if unhealthy, along with a plain text description of why it is unhealthy).
```

Comment:
```suggestion
Optionally, the ``canton`` process can expose an HTTP endpoint indicating whether the process believes it is healthy. This may be used as an uptime check or as a `Kubernetes liveness probe <https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/>`__. If enabled, the ``/health`` endpoint will respond to a ``GET`` HTTP request with a 200 HTTP status code (if healthy) or 500 (if unhealthy, along with a plain text description of why it is unhealthy).
```